lib/jellyfish/component.ex

```diff
@@ -0,0 +1,32 @@
+defmodule Jellyfish.Component do
+  @moduledoc """
+  Defines `t:Jellyfish.SDK.Component.t/0`.
+  """
+
+  @enforce_keys [
+    :id,
+    :type
+  ]
+  defstruct @enforce_keys
+
+  @typedoc """
+  Id of a component in a form of UUID string.
```

Comment:
A bit too specific I think. It's also a little... revealing. This is the SDK mind you, it just makes API calls and it shouldn't care about server implementation details for our own sanity. We're going to have a whole bunch of these, and correcting server implementation details in docs for 20 SDKs doesn't sound like fun. But I think we should mention that the peer id is unique in the <room/JF (?)> scope.
```suggestion
  Id of the component, unique across every room
```